src/journey-maps/angular/journey-maps.ts

```diff
@@ -300,6 +304,24 @@ export class SbbJourneyMaps implements OnInit, AfterViewInit, OnDestroy, OnChang
     };
   }
 
+  private _homeButtonOptions: SbbViewportOptions = this._defaultHomeButtonOptions;
```

Comment:
I don't think this should be of type `SbbViewportOptions`.

See these open changes here: https://github.com/sbb-design-systems/sbb-angular/pull/1479/files#diff-090b8998e572b3694f02c5c7708aa34d10b0101b0c07c6852579c1fc72f85a50

The new properties don't relate to the home button.

Reply:
I reverted my previous commit after reading your comment.